docs/root/version_history/current.rst

```diff
@@ -20,6 +20,7 @@ Bug Fixes
 *Changes expected to improve the state of the world and are unlikely to have negative effects*
 
 * http: raise max configurable max_request_headers_kb limit to 8192 KiB (8MiB) from 96 KiB in http connection manager.
+* listener: fix the crash when a filter chain only listener update is superseded by the listener removal or full listener update.
```

Comment:
What does supersede mean?

Reply:
by the next listener update/removal at the same listener name?